### PR TITLE
Add tip about turning off WholeStageCodeGen to split_multiallelics transformer doc

### PR DESCRIPTION
**Pull request by kianfar77**
## What changes are proposed in this pull request?
- Adds a tip to the `split_multiallelics` transformer documentation about turning off the whole stage code generation for performance. Modifies the notebook to reflect the tip. 

- Adds a sentence on how splitter behaves on unflattened info fields. 

## How is this patch tested?
- [ ] Unit tests
- [ ] Integration tests
- [x] Manual tests

(Details)

### CODE REVIEW

File: docs/source/etl/variant-normalization.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Glow provides the ``normalize_variants`` transformer to be applied on a variant
...
+  The variant normalization algorithm used by the ``normalize_variants`` transformer follows the same logic as the one used in normalizations tools such as `bcftools norm <http://www.htslib.org/doc/bcftools.html#norm>`_ and `vt normalize <https://genome.sph.umich.edu/wiki/Vt#Normalization>`_ tools. This normalization logic is different from the one used by GATK's `LeftAlignAndTrimVariants <https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_variantutils_LeftAlignAndTrimVariants.php>`_, which sometimes yields incorrect normalization (see `Variant Normalization <https://genome.sph.umich.edu/wiki/Variant_Normalization>`_ for more details).
```
Comment (karenfeng) [inline docs/source/etl/variant-normalization.rst]: Nit: the last word in the first sentence ("tools") is extraneous.

File: docs/source/etl/variant-splitter.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Split Multiallelic Variants
...
+    - Each ``INFO`` field is appropriately split among split rows if it has the same number of elements as number of ``ALT`` alleles, otherwise it is repeated in all split rows. The boolean ``INFO`` field ``splitFromMultiAllelic`` is added/modified to reflect whether the new row is the result of splitting a multiallelic row through this transformation or not. A new ``INFO`` field called ``OLD_MULTIALLELIC`` is added to the DataFrame, which for each split row, holds the ``CHROM:POS:REF/ALT`` of its original multiallelic row. Note that the ``INFO`` field must be flattened (as explained :ref:`here<vcf>`) in order to be split by this transformer. Unflattened ``INFO`` fields inside the ``attributes`` field will not be split, but just repeated in whole across all split rows.
```
Comment (karenfeng) [inline docs/source/etl/variant-splitter.rst]: As we plan to phase out VCFRows (the only reason `attributes` exists), we could reword this to be forward-compatible. For example, "Unflattened ``INFO`` fields (such as those inside an ``attributes`` field)".

File: docs/source/etl/variant-splitter.rst [hunk 3]
```diff
@@ -70,5 +70,10 @@ Assuming ``df_original`` is a variable of type DataFrame which contains the geno
 
             df_split = Glow.transform("split_multiallelics", df_original)
 
+.. tip::
+
+    The ``split_multiallelics`` transformer is often significantly faster if the `whole-stage code generation` feature of Spark Sql is turned off. Therefore, it is recommended that you turn off this feature using the command ``spark.conf.set("spark.sql.codegen.wholeStage", False)`` before using this transformer.
```
Comment (karenfeng) [inline docs/source/etl/variant-splitter.rst]: This is language-specific, right? If so we should have a Python and a Scala version.

File: docs/source/etl/variant-normalization.rst (hunk 1 above)
Comment (kianfar77) [reply docs/source/etl/variant-normalization.rst]: removed

File: docs/source/etl/variant-splitter.rst (hunk 2 above)
Comment (kianfar77) [reply docs/source/etl/variant-splitter.rst]: changed

File: docs/source/etl/variant-splitter.rst (hunk 3 above)
Comment (kianfar77) [reply docs/source/etl/variant-splitter.rst]: tabs created.

Comment (karenfeng) [reply docs/source/etl/variant-splitter.rst]: Looks good!

Comment (karenfeng) [reply docs/source/etl/variant-splitter.rst]: Maybe it would be good to remind the user to turn codegen back on after as well.

Comment (karenfeng) [reply docs/source/etl/variant-splitter.rst]: Maybe just say `that you temporarily turn off`

Comment (kianfar77) [reply docs/source/etl/variant-splitter.rst]: Added remember warning
